### PR TITLE
Fix for UserAccount casting error in 1.0.1+48 release

### PR DESCRIPTION
**Pull request by dessap**
### Change description ###
Fix for UserAccount casting error in 1.0.1+48 release

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: AdminWebsite/AdminWebsite.AcceptanceTests/Steps/DashboardSteps.cs [hunk 1]
```diff
@@ -21,7 +22,7 @@ public DashboardSteps(Dashboard dashboard, ScenarioContext injectedContext)
         public void ThenBookAVideoHearingPanelIsDisplayed(string panelText)
         {
             var panels = _dashboard.VhPanelTitle();
-            switch (_scenarioContext.Get<string>("User"))
+            switch (_scenarioContext.Get<UserAccount>("User").Role)
```
Comment (rehmanab) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Steps/DashboardSteps.cs]: If _scenarioContext.Get<UserAccount>("User") is null, then .Role will throw an exception

File: AdminWebsite/AdminWebsite.AcceptanceTests/Steps/HearingDetailsSteps.cs [hunk 2]
```diff
@@ -90,7 +91,7 @@ public void SelectQuestionnaireNotRequired()
         [Then(@"case type dropdown should not be populated")]
         public void ThenCaseTypeDropdownShouldNotBePopulated()
         {
-            switch (_scenarioContext.Get<string>("User"))
+            switch (_scenarioContext.Get<UserAccount>("User").Role)
```
Comment (rehmanab) [inline AdminWebsite/AdminWebsite.AcceptanceTests/Steps/HearingDetailsSteps.cs]: If _scenarioContext.Get<UserAccount>("User") is null, then .Role will throw an exception

Comment (dessap) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Steps/HearingDetailsSteps.cs]: done for next commit

File: AdminWebsite/AdminWebsite.AcceptanceTests/Steps/DashboardSteps.cs (hunk 1 above)
Comment (dessap) [reply AdminWebsite/AdminWebsite.AcceptanceTests/Steps/DashboardSteps.cs]: done for next commit

**Pull request merged**